org.eclipse.wildwebdeveloper/pom.xml

```diff
@@ -132,6 +132,9 @@
 					</execution>
 				</executions>
 			</plugin>
+
```

Comment:
No need for extra lines here. You can reset to previous state to simplify the change (and the review)